### PR TITLE
Updating Store react api material UI version

### PR DESCRIPTION
**Pull request by dushansilva**
This PR fixes https://github.com/wso2/product-apim/issues/5694 issue.

This also fixes minor UX issues in APIM store

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/InlineMessage.jsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ class InlineMessage extends React.Component {
         const { classes, type } = this.props;
         const messgeType = type || 'info';
         return (
-            <Paper className={classes.root} elevation={1} {...this.props}>
+            <Paper elevation={2} className={classes.root} elevation={1} {...this.props}>
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/InlineMessage.jsx]: there are 2 elevation props here shall we remove one

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx [hunk 2]
```diff
@@ -360,7 +360,7 @@ class ApiConsole extends React.Component {
                 <Grid container className={classes.grid}>
                     {!user && (
                         <Grid item md={6}>
-                            <Paper className={classes.userNotificationPaper}>
+                            <Paper elevation={2} className={classes.userNotificationPaper}>
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx]: Do we need to set `elevation={2}` specifically here ?

Comment (dushansilva) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/ApiConsole/ApiConsole.jsx]: according to [1] the default elevation in v3 is similar to elevation=2 in v4

[1] - https://material-ui.com/guides/migration-v3/

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/InlineMessage.jsx (hunk 1 above)
Comment (dushansilva) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/InlineMessage.jsx]: fixed https://github.com/wso2/carbon-apimgt/pull/7147/commits/158d1d591856515f2507b662e9cf02ae24bb9d74

**Pull request merged**